modules/odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-yaml-view.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assembly:
+//
+// * web_console/customizing-the-web-console.adoc
+
+:_content-type: PROCEDURE
+
+[id="odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-yaml-view_{context}"]
+= Customizing a Developer catalog or it's sub-catalogs using the YAML view
+
+.Prerequisites
+
+* You must have administrator privileges.
+
+.Procedure
+
+. In the *Administrator* perspective of the web console, navigate to *Administration* -> *Cluster Settings*.
+. Select the *Configuration* tab, click the *Console (operator.openshift.io)* resource and see the *Details* page.
+. Click the *YAML* tab to open the editor and edit it as per your requirements. 
+.. For example, to disable a Developer sub-catalog, insert the following snippet for 'Defining a list of disabled Developer catalog resources'.
+
++ 
+[source,yaml]
+----
+``yaml
+apiVersion: operator.openshift.io/v1
+kind: Console
+metadata: 
+  name: cluster
+  ...
+spec:
+  customization:
+    developerCatalog:
+      categories:
+      types:
+        state: Disabled
+        disabled:
+          - BuilderImage
+          - Devfile
+          - HelmChart
+----
+
+. Click *Save*.
+
+[Note]
+====
+By default, the Developer sub-catalogs are enabled in the Administrator view of the Web Console.
+====
+
+The following snippets can be used in the YAML editor for a variety of scenarios:
+
+Use the following snippet to display all the sub-catalogs by default by setting the _state_ type as *Enabled*.
+[source,yaml]
+----
+spec:
+  customization:
+    developerCatalog:
+      categories:
+      types:
+        state: Enabled
+----
+
+Similarly, use the following snippet to disable the full Developer catalog and all sub-catalogs by setting the _state_ type as *Disabled*:
+[source,yaml]
+----
+spec:
+  customization:
+    developerCatalog:
+      categories:
+      types:
+        state: Disabled
+----
+
+You can use the following snippet when a cluster administrator defines a list of sub-catalogs, which are disabled in the Web Console.
+[source,yaml]
+----
+spec:
+  customization:
+    developerCatalog:
+      categories:
+      types:
+        state: Disabled
+        disabled:
+          - BuilderImage
+          - Devfile
+          - HelmChart
+          - ...
+----
```

Comment:
```suggestion
You can use the following snippet when a cluster administrator defines a list of sub-catalogs, which are enabled in the Web Console.
[source,yaml]
----
spec:
  customization:
    developerCatalog:
      categories:
      types:
        state: Enabled
        enabled:
          - BuilderImage
          - Devfile
          - HelmChart
          - ...
----
```

A snippet for disabling sub-catalog types is already provided at line no: 24

Reply:
Fixed.